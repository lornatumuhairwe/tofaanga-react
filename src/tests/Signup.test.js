import React from 'react';
import renderer from 'react-test-renderer';
import SignupForm from '../components/Authentication/Signup';

describe('Signup snapshot', () => {
  it('renders snapshot appropriately', () => {
    const signup = renderer.create(<SignupForm />);
    expect(signup.toJSON()).toMatchSnapshot();
  });
});
