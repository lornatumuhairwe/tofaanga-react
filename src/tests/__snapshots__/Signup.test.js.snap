// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup snapshot renders snapshot appropriately 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-6 col-md-4 col-md-offset-4"
    >
      <h1
        className="text-center login-title"
      >
        Sign up start keeping track of your bucketlists 
      </h1>
      <div
        className="account-wall"
      >
        <div
          className="notifications-wrapper"
          style={Object {}}
        />
        <form
          className="form-signin"
          onSubmit={[Function]}
        >
          <input
            autoFocus={true}
            className="form-control"
            id="name"
            onChange={[Function]}
            placeholder="Name"
            required={true}
            type="text"
            value=""
          />
          <input
            autoFocus={true}
            className="form-control"
            id="email"
            onChange={[Function]}
            placeholder="Email"
            required={true}
            type="text"
            value=""
          />
          <input
            autoFocus={true}
            className="form-control"
            id="bd"
            onChange={[Function]}
            placeholder="Birth Date"
            required={true}
            type="date"
            value=""
          />
          <input
            className="form-control"
            id="pwd"
            onChange={[Function]}
            placeholder="Password"
            required={true}
            type="password"
            value=""
          />
          <button
            className="btn btn-lg btn-primary btn-block btn btn-default"
            disabled={false}
            type="submit"
          >
            Sign up
          </button>
          <span
            className="clearfix"
          />
        </form>
      </div>
      <a
        className="text-center new-account"
        href=""
        onClick={undefined}
      >
        Already have an account? Sign in
      </a>
    </div>
  </div>
</div>
`;
